astronomer/providers/amazon/aws/example_dags/example_s3.py

```diff
@@ -1,15 +1,37 @@
-from datetime import timedelta
+import os
+from datetime import datetime, timedelta
 
-from airflow.models.dag import DAG
-from airflow.utils.timezone import datetime
+from airflow import DAG
+from airflow.providers.amazon.aws.operators.s3 import (
+    S3CreateBucketOperator,
+    S3DeleteBucketOperator,
+)
+from airflow.providers.amazon.aws.transfers.local_to_s3 import (
+    LocalFilesystemToS3Operator,
+)
 
-from astronomer.providers.amazon.aws.sensors.s3 import S3KeySensorAsync
+from astronomer.providers.amazon.aws.sensors.s3 import (
+    S3KeySensorAsync,
+    S3KeySizeSensorAsync,
+    S3KeysUnchangedSensorAsync,
+)
 
 default_args = {
     "retry": 5,
     "retry_delay": timedelta(minutes=1),
 }
 
+PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "astronomer-airflow-providers")
+S3_BUCKET_NAME = os.environ.get("S3_BUCKET_NAME", "test-bucket")
+S3_BUCKET_KEY = os.environ.get("S3_BUCKET_KEY", "sample_key.txt")
+S3_BUCKET_WILDCARD_KEY = os.environ.get("S3_BUCKET_WILDCARD_KEY", "sam*")
+WILDCARD_KEY_FALSE = os.environ.get("WILDCARD_KEY_FALSE", False)
+WILDCARD_KEY_TRUE = os.environ.get("WILDCARD_KEY_TRUE", True)
+PREFIX = os.environ.get("PREFIX", "test")
+INACTIVITY_PERIOD = os.environ.get("INACTIVITY_PERIOD", 5)
+REGION_NAME = os.environ.get("REGION_NAME", "us-east-2")
+LOCAL_FILE_PATH = os.environ.get("LOCAL_FILE_PATH", "/usr/local/airflow/sample_key.txt")
```

Comment:
How can this local file be present in /usr/local/airflow always. I think we should include a sample file in our repo and then copy that file over to S3 with one of the operator, as part of this DAG.

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/86/commits/93d4a9ec4f724b29a84a50c66fe1c1b51347cfe8) I have added `sample_key.txt` in the `example_dags` folder